### PR TITLE
Added documentation about travis token

### PR DESCRIPTION
**Pull request by agnivade**
/cc @waldyrious

### CODE REVIEW

**Comment by waldyrious** [comment]
Nice :+1:

Please also mention that the upload is done via https://github.com/tldr-pages/tldr/blob/master/scripts/build.sh#L31, and add a comment to that file as well with similar documentation.

**Comment by waldyrious** [comment]
For future reference, this PR is a follow-up to #1899.

**Comment by agnivade** [comment]
@waldyrious - PTAL

File: scripts/build.sh [hunk 1]
```diff
@@ -28,6 +28,7 @@ function build_archive {
 }
 
 function upload_assets {
+  # ${GH_TOKEN} is referred from the secure variable inside .travis.yml
```
Comment (waldyrious) [inline scripts/build.sh]: How about "referred from the secure variable inside .travis.yml" --> "defined as a secure variable in .travis.yml"?

Comment (waldyrious) [reply scripts/build.sh]: Also, I'd add a comment at the top of this file, saying something like:

    # This script is executed by Travis CI when a PR is merged (i.e. in the `after_success` step).

Comment (agnivade) [reply scripts/build.sh]: Done

**Pull request merged**